src-electron/util/zcl-util.js

```diff
@@ -440,7 +440,7 @@ async function dataTypeCharacterFormatter(
 function isEnum(db, enum_name, packageId) {
   return queryZcl
     .selectEnumByName(db, enum_name, packageId)
-    .then((enums) => (enums ? dbEnum.zclType.enum : dbEnum.zclType.unknown))
+    .then((enums) => (enums ? true : false))
```

Comment:
Or could do:
```suggestion
    .then((enums) => (Boolean(enums)))
```
but either way.